_posts/2016-01-23-hair-xray.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: X-rays show hair treatments don't penetrate or fix hair
+author: neuroamanda
+---
+
+_But permanent waving does alter protein structure inside hair_
+
+Miracle products for fussy hair have probably been around longer than hair has even been studied.
+Though hair structure has now been investigated in great detail, with tools like X-ray diffraction and electron microscopy, hair repair products proliferate, and their benefits can be dubious at best.
+Beyond settling this cosmetic question, however, studying hair with imaging methods can be instructive for science &mdash; and health.
+Abnormally curly or twisted hair can actually be indicative of an underlying disease state; [breast cancer](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3086309/), for example, can be accurately predicted using hair samples.
+Disease screening thus might benefit from hair imaging, but it's only useful to study hair if the frequently used hair care products &mdash; shampoo, conditioner, and permanent waving treatments &mdash; don't affect internal hair structure.
+
+In an [earlier study on hair structure](https://doi.org/10.7717/peerj.619), researchers sampled hair from a variety of individuals who used different amounts of shampoo and conditioner.
+In their new study, they wanted to [study just one person's hair](https://doi.org/10.7717/peerj.1296), to discount any effects of differential product use.
+
+Shampoos and conditioners have a whole host of ingredients, many of them completely unnecessary for their primary functions: cleaning and smoothing.
+Both products mainly interact with hair at the surface, or cuticle, level. Shampoo and conditioner will only reach the next layer, the cortex, if the cuticle is very damaged, in which case shampooing may contribute to even _more_ hair damage.
+Repeated shampooing extracts more hair surface lipids (oils) while conditioning retains ingredients on the hair surface.
+Both products can affect the X-ray diffraction signal when the hair is imaged.
+Perming, for its part, also induces major structural changes in hair, breaking and re-forming chemical bonds within hair and making the hair surface more likely to shed lipids during shampooing.
+
+![False-color scanning electron micrograph of a bleached and straightened strand of hair.]({{ site.url }}/images/2016-01-23-hair-xray.jpg)
+
+> _False-color scanning electron micrograph of a bleached and straightened strand of hair. [Image](https://www.flickr.com/photos/wellcomeimages/15854964238/) by Wellcome Images / [CC BY-NC-ND 2.0](https://creativecommons.org/licenses/by-nc-nd/2.0/)_
+
+One female participant volunteered her hair for the shampoo, conditioner, and perming experiments in this study.
```

Comment:
One place where the serial comma actually confuses things: You want to say "shampoo experiments", "conditioner experiments" and "perming experiments", but it comes across as "shampoo", "condition", and "perming experiments".